style.css

```diff
@@ -465,143 +498,98 @@ body {
   color: #5e6c84;
 }
 
-@media (max-width: 500px) {
-  .header {
-    width: 37.5rem;
+@media (min-width: 60rem) {
+  .hamburger-menu {
+    display: none;
   }
 
-  .text {
-    width: 37.5rem;
-    height: 60.4rem;
+  .nav-items ul {
     display: flex;
-    justify-content: center;
-    align-items: center;
-    padding: 3rem 0;
+    gap: 0.8rem;
+    padding-right: 14.2rem;
   }
 
-  .text-section {
-    width: 32.7rem;
+  .nav-links {
+    font-family: Poppins, sans-serif;
+    font-weight: 500;
+    font-size: 1.5rem;
+    line-height: 2rem;
+    color: #344563;
+    text-transform: capitalize;
   }
 
-  .work {
-    width: 37.5rem;
-    padding: 11.4rem 2.4rem;
+  .logo {
+    padding-left: 14.2rem;
   }
 
-  .card-work {
+  .text {
     display: flex;
-    flex-direction: column;
-    align-items: center;
-    padding: 1.6rem;
-    gap: 1.2rem;
-    border: 1px solid #dfe1e6;
-    border-radius: 1.6rem;
-  }
-
-  .snapshot-portfolio {
-    width: 29.5rem;
-    height: 22rem;
-    padding: 0;
-  }
-
-  .snapshot-portfolio img {
-    width: 100%;
-    height: 100%;
-    object-fit: cover;
+    justify-content: center;
+    height: 84.6rem;
   }
 
-  .left-block {
-    display: flex;
-    flex-direction: column;
-    align-items: flex-start;
-    padding: 0;
-    gap: 1.2rem;
-    width: 29.5rem;
+  .head-section {
+    font-size: 4.8rem;
   }
 
-  .work-title {
-    font-size: 4rem;
-    line-height: 4.4rem;
-    color: #172b4d;
+  .para-section {
+    font-size: 2rem;
   }
 
-  .frame {
-    font-size: 1.3rem;
-    line-height: 1.6rem;
+  .work {
+    min-width: 60rem;
   }
 
-  .frame .client {
-    color: #344563;
-    font-weight: 600;
+  .card-work {
+    display: grid;
+    grid-template-columns: 54.4rem 1fr;
   }
```

Comment:
- [ ] In the figma design there is margin on both sides of the cards. but in your design at 1024px width, there is no margin even the card disappears from both sides.this is against the principle of responsive design which is one of the core requirement. kindly make sure to have the exact margins on both sides and at all screen sizes to make it look nicer.
your design
![asdf](https://user-images.githubusercontent.com/96683843/213249139-648b37c2-3a24-43a6-b969-272deb254aff.jpg)
figma design
![fd](https://user-images.githubusercontent.com/96683843/213249239-cdb6b51f-6ba3-40a8-a274-b8e9c64fc3ba.jpg)